src/sonic-yang-models/yang-models/sonic-feature.yang

```diff
@@ -66,7 +66,38 @@ module sonic-feature{
                     type stypes:feature_state;
                     default "disabled";
                 }
+                
+                leaf check_up_status {
+                    description "This configuration controls the system ready tool to check
+                                the app ready/up status";
+                    type boolean;
+                    default false;
+                }
+            }
+        }
+        container HOST_FEATURE {
```

Comment:
i am not sure we should introduce such yang model, why not in the feature table, we add a flag to indicate if it is host or container feature.